autometa/taxonomy/ncbi.py

```diff
@@ -111,7 +111,8 @@ def name(self, taxid, rank=None):
             Name of provided `taxid` if taxid is found in names.dmp else will return None
 
         """
-        self.is_valid_taxid(taxid)
+        if not self.is_valid_taxid(taxid):
+            return False
```

Comment:
The docstring notes the ReturnTypes as `str` or `None`, not `bool` which is returned if `is_valid_taxid` is False